maps/sccv_horizon/code/sccv_horizon_overmap.dm

```diff
@@ -175,3 +194,42 @@
 	icon_keyboard = null
 	circuit = null
 
+//
+// ------------------------------------------- quark
+//
+
+/obj/effect/overmap/visitable/ship/landable/quark
+	name = "Quark"
+	class = "SCCV"
+	designation = "Quark"
+	desc = "..."
+	shuttle = "Quark"
+	icon_state = "pod" // <-------------- temporary, hopefully someone sprites it some day
+	moving_state = "pod_moving"
+	colors = list("#cfd4ff", "#78adf8")
+	// scanimage = "quark.png" // <-------------- hopefully someone sprites it some day
+	designer = "Hephaestus Industries, NanoTrasen"
+	volume = "14 meters length, 7 meters beam/width, 5 meters vertical height"
+	weapons = "Extruding fore-mounted low-caliber ballistic rotary armament"
+	sizeclass = "Jester-type Scout Skiff"
```

Comment:
The Canary is a Jester class, gonna want adjusting here.

Reply:
fixed